module/VuFind/src/VuFind/Config/PathResolver.php

```diff
@@ -137,6 +139,64 @@ public function getLocalConfigLocation(
         return $currentLocation;
     }
 
+    /**
+     * Get all configuration locations in a specific path.
+     *
+     * @param string $path Path of the directory to scan
+     *
+     * @return ConfigLocationInterface[]
+     */
+    public function getConfigLocationsInPath(string $path): array
+    {
+        $dirContent = is_dir($path) ? scandir($path) : [];
+        $result = [];
+        foreach ($dirContent as $item) {
+            // Exclude "." and "..".
+            $ignorePattern = "/^\.{1,2}$/";
+            if (
+                preg_match($ignorePattern, $item)
+            ) {
+                continue;
+            }
+            $itemPath = $path . DIRECTORY_SEPARATOR . $item;
+            $configLocation = $this->getConfigLocationOnPath($itemPath);
+            // ignore locations without a matching handler
+            if ($configLocation === null || $this->configHandlerManager->getForLocation($configLocation) === null) {
```

Comment:
I think it would be useful to have another method such as `$this->configHandlerManager->hasForLocation($configLocation)` that would just check for the plugin instead of getting it. Even if it's a shared object, getting it does much more than just `->has()`.